main.tf

```diff
@@ -111,6 +111,7 @@ check "runtime_support" {
         "python3.11",
         "python3.12",
         "python3.13",
+        "provided.al2023",
```

Comment:
Forgot to mention this earlier, while we're here, let's also add `provided.al2` (see in [list](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported) of supported lambda runtimes). No need to make separate examples, we can just add it to this check